src/main/java/com/yevhenberladyniuk/debts/dto/PartnerDto.java

```diff
@@ -21,4 +22,9 @@
     @NotBlank
     private String lastName;
 
+    @NotBlank
+    private double debt;
+
+    @NotBlank
```

Comment:
`@NotBlank` is used for `String`
By the way, there's no needs of these constraints in DTO that is returned from server

Reply:
Is this class used somewhere?

Reply:
yes, inside the ```PartnerService.updateById``` method